matlab/CreateBatchPlots.m

```diff
@@ -1,31 +1,100 @@
-function [ ] = CreateBatchPlots( summary_data, units )
+function [ ] = CreateBatchPlots( summary_data, ...
+    units, config )
 %CREATEBATCHPLOTS Batch creation of plots.
-%   The summary_data parameter is a struct of the simulation summary data.
-%   The units parameter is a struct with mappings from properties to units.
+%   The summary_data parameter is a struct of the 
+%   simulation summary data.
+%   The units parameter is a struct with mappings 
+%   from properties to units.
 
 % As user where to save plots
 folder = uigetdir('','Select directory to save the plots in');
 xdata = summary_data.('FIELD').('TIME');
 
-fpr(1);
-wgpr(2);
-wlpr(2);
-wopr(2);
-wwpr(2);
-wgpt(3);
-wlpt(3);
-wopt(3);
-wwpt(3);
-wbhp(4);
-wwct(5);  
-fgpr(6);
+if isunix
+    system(['cd ' folder '; rm -f *.pdf ;']);
+end
```

Comment:
This entire section should probably be moved to a separate function. It could, for example, take as input the summary_data object, and return a list of function handles that can be executed in a for-loop.